CONTRIBUTING.md

```diff
@@ -50,54 +50,56 @@ Codespaces is new technology that allows you to use a container as your developm
 1. From the Azure SDK GitHub repo, click on the "Code -> Open with Codespaces" button.
 1. Open a Terminal. The development environment will be ready for you. Continue to [Building and Testing](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#building-and-testing).
 
-#### VS Code Codespaces
+### VS Code Codespaces
 
 1. Install the [VS Code Remote Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
 1. When you open the Azure SDK for C++ repo in VS Code, it will prompt you to open the project in the Dev Container. If it does not prompt you, then hit CTRL+P, and select "Remote-Containers: Open Folder in Container..."
 1. Open a Terminal. The development environment will be ready for you. Continue to [Building and Testing](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#building-and-testing).
 
-### Full Local Setup
+## Full Local Setup
 
-#### Pre-requisites
+### ** Pre-requisites **
 
-##### CMake
+#### -> CMake
 
 CMake version 3.13 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
-##### Third Party Dependencies
+### Third Party Dependencies
 
-- curl
-- libxml2
-- clang-format (min version 10)
+Azure SDK use Vcpkg manifest mode to declare the [list of required dependencies](https://github.com/Azure/azure-sdk-for-cpp/blob/main/vcpkg.json) for building the SDK service libraries. It will also get and set up Vcpkg automatically. You can move on to [Building the project](#building-the-project) and skip next part if you are not interested in learning the alternatives for setting up dependencies.
 
-Vcpkg can be used to install the Azure SDK for CPP dependencies into a specific folder on the system instead of globally installing them.
-Follow [vcpkg install guide](https://github.com/microsoft/vcpkg#getting-started) to get vcpkg and install the following dependencies:
+If you didn't jump to `Build the project`, you will now learn about how you can control Vcpkg and dependencies integration.
 
-```sh
-./vcpkg install curl libxml2
-```
+#### Setting up Vcpkg automatic integration
 
-When using vcpkg, you can set the `VCPKG_ROOT` environment variable to the vcpkg Git repository folder. This would automatically set the CMake variable `CMAKE_TOOLCHAIN_FILE` for you, enabling the project to use any library installed with vcpkg.
+If the CMake option _-DCMAKE_TOOLCHAIN_FILE=..._ is not defined to generate the project, the Azure SDK project will automatically get Vcpkg and link it to get its dependencies. You can use the next environment variables to change this behavior:
```

Comment:
nit: Am I supposed to define the `DCMAKE_TOOLCHAIN_FILE` option if I want to customize the vcpkg integration? How am I as a contributor supposed to see if this is defined or being used or not.

The table below doesn't mention that, so you may want to keep the info about how VCPKG_ROOT interacts with `CMAKE_TOOLCHAIN_FILE` on behalf of the user.